pyblish_royalrender/plugins/validate_royalrender.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+import pyblish.api
+
+
+class PyblishRoyalRenderValidate(pyblish.api.ContextPlugin):
+    """ Validates the installation of Royal Render. """
+
+    order = pyblish.api.ValidatorOrder
+    label = "Royal Render"
+
+    def process(self, context):
+
+        msg = "\"RR_Root\" environment variable could not be found."
+        assert "RR_Root" in os.environ, msg
```

Comment:
In the python script? Got newline here, don't know why its not showing on github.